src/styles/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: container-width($square-size, 7, $sides-paddings);
+  padding: 10px;
```

Comment:
if I understand correctly, then the top paddings should be larger according to p. 11 of the check list and you can set padding with using variables;